### PR TITLE
adding test file

### PR DESCRIPTION
**Pull request by prabhaskulk**
this is just testing

### CODE REVIEW

**Review by venukulk — COMMENTED** [review]
this is python binary file

File: stock_price_lstm.ipynb
Comment (venukulk) [inline stock_price_lstm.ipynb:1]: this is binary

Comment (prabhaskulk) [reply stock_price_lstm.ipynb:1]: it is jupyter notebook. i will check on this

**Review by venukulk — APPROVED** [review]
Approved

**Pull request merged**